dev_tasks/yard.rake

```diff
@@ -8,9 +8,7 @@ API_DIR = File.join(SITE_DIR, 'api', 'cucumber-rails', 'ruby', 'yardoc')
 
 namespace :api do
   file :dir do
-    unless File.directory?(SITE_DIR)
-      raise "You need to git clone git@github.com:cucumber/cucumber.github.com.git #{SITE_DIR}"
-    end
+    raise "You need to git clone git@github.com:cucumber/cucumber.github.com.git #{SITE_DIR}" unless File.directory?(SITE_DIR)
```

Comment:
This is where RuboCop does the wrong thing due to line length.

(Once upon a time there was a separate setting for this particular cop, independent of line length. The more I think about it, the more I'm convinced this setting should return.)

Reply:
@mvz Thanks, this is a great example of a 122-character line that isn't too long. I wouldn't want to break up the string literal just for the sake of breaking it up.

Reply:
@marnen the string literal isn't the reason this line is so long. In the original, the longest line (12) was 95 characters. In any case, I'd still consider splitting it up since it has actual code at the end (`SITE_DIR`).

Reply:
...whereas for me, doing so would be a perfect example of pointless use of character count. :) I think it’s fine as it stands.